tests/uses/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.c
```

Comment:
Unfortunately the include paths are absolute at the moment (yuck!) but that is a good idea for the future.